examples/config-scrub.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.1.0-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot"
+    },
+    "http": {
+        "address": "127.0.0.1",
+        "port": "8080"
+    },
+    "log": {
+        "level": "debug"
+    },
+    "extensions": {
+        "scrub": {
```

Comment:
Another option is to put this under the "storage" section. But let's keep it here for now.